src/views/confirmationMail.pug

```diff
@@ -11,7 +11,7 @@ html
             |
             | aktiv.
         p
-            div Um das Angebot zu verlängern, klicken Sie bitte auf den folgenden Link:
+            div Um das Angebot zu erneuern, klicken Sie bitte auf den folgenden Link:
```

Comment:
Is 'Um die Dauer Ihres Angebots zu erneuern, …' better?

Reply:
Hmm... I don't think so. I better like `Angebot erneuern` :D